lib/class-wp-rest-image-editor-controller.php

```diff
@@ -36,35 +42,88 @@ public function __construct() {
 	public function register_routes() {
 		register_rest_route(
 			$this->namespace,
-			$this->rest_base . '/apply',
+			$this->rest_base,
 			array(
 				array(
 					'methods'             => WP_REST_Server::EDITABLE,
 					'callback'            => array( $this, 'apply_edits' ),
 					'permission_callback' => array( $this, 'permission_callback' ),
 					'args'                => array(
-						'x'        => array(
-							'type'     => 'float',
-							'minimum'  => 0,
-							'required' => true,
-						),
-						'y'        => array(
-							'type'     => 'float',
-							'minimum'  => 0,
-							'required' => true,
-						),
-						'width'    => array(
-							'type'     => 'float',
-							'minimum'  => 1,
-							'required' => true,
-						),
-						'height'   => array(
-							'type'     => 'float',
-							'minimum'  => 1,
-							'required' => true,
-						),
-						'rotation' => array(
-							'type' => 'integer',
+						array(
+							'modifiers' => array(
+								'type'     => 'array',
+								'required' => true,
+								'items'    => array(
+									'anyOf' => array(
```

Comment:
Core does not support the `anyOf` keyword.

Reply:
Hmm, so that's going to make validating this more challenging. What JSON Schema version is it using?

Reply:
v4 but we don't support the entirety of the spec, see `rest_validate_value_from_schema`.